vault/provider.go

```diff
@@ -864,6 +865,48 @@ func ReadContextWrapper(f schema.ReadContextFunc) schema.ReadContextFunc {
 	}
 }
 
+func VersionCheckWrapper(f schema.CreateFunc, d *schema.ResourceDiff, meta interface{}, minVersion string) schema.CreateFunc {
```

Comment:
Is this being used?

Reply:
I thought we might need a wrapper for the resources using the deprecated functions, but now that I think more about it, semantic version comparisons will mostly be used for new features which won't use this wrapper function and any old resources that need semantic version comparison should probably be updated to not use deprecated code anyway. Removed this in the latest commits